src/main/scala/com/databricks/labs/overwatch/pipeline/Schema.scala

```diff
@@ -1062,7 +1062,11 @@ object Schema extends SparkSessionWrapper {
     StructField("enable_unsafe_SSL", BooleanType, nullable = true),
     StructField("thread_pool_size", IntegerType, nullable = true),
     StructField("api_waiting_time", LongType, nullable = true),
-    StructField("mount_mapping_path", StringType, nullable = true)
+    StructField("mount_mapping_path", StringType, nullable = true),
+    StructField("eh_conn_string", StringType, nullable = true),
+    StructField("aad_tenant_id", StringType, nullable = true),
+    StructField("aad_client_id", StringType, nullable = true),
+    StructField("aad_client_secret_key", StringType, nullable = true)
```

Comment:
we also support AAD authority endpoint (the `aad_authority_endpoint` in the old implementation`) so we can use it not only on "Normal Azure" but also in the GovCloud, China & German Clouds

Reply:
as discussed no code changes needed

Reply:
sorry. Maybe I explained incorrectly - the `aad_authority_endpoint` is necessary if we want to support other Azure clouds: Azure GovCloud, Azure China, Azure Germany

Reply:
no. authority endpoint isn't a part of the EH connection string. it's an URL of the AAD REST API that will be used to generate AAD tokens